action/QuadMove.action

```diff
@@ -0,0 +1,11 @@
+#goal definition
+string movement_type
+bool preempt
+float64 takeoff_height
+---
+#result definition
+bool success
```

Comment:
Just realized I didn't mean preempted, I actually meant rejected.  Although I think preemption could probably be returned as well, that could be useful for the gameplay algorithm.  Either way, I think this can stay for now and go into another PR if we want to add that functionality.